custom-video-player/script.js

```diff
@@ -4,6 +4,8 @@ const stop = document.getElementById('stop');
 const progress = document.getElementById('progress');
 const timestamp = document.getElementById('timestamp');
 
+var a = foo
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/fjrdomingues/vanillawebprojects/pullRequest?prid=9164958)
```suggestion
var a = foo;
```